src/common/syncjournaldb.cpp

```diff
@@ -42,16 +42,18 @@
 #define IS_PREFIX_PATH_OR_EQUAL(prefix, path) \
     "(" path " == " prefix " OR " IS_PREFIX_PATH_OF(prefix, path) ")"
 
+namespace {
+const auto getFileRecordQueryC = QByteArrayLiteral("SELECT path, inode, modtime, type, md5, fileid, remotePerm, filesize,"
+                                                   " ignoredChildrenRemote, contentchecksumtype.name || ':' || contentChecksum,"
+                                                   " hasDirtyPlaceholder"
+                                                   " FROM metadata"
+                                                   " LEFT JOIN checksumtype as contentchecksumtype ON metadata.contentChecksumTypeId == contentchecksumtype.id ");
```

Comment:
Maybe add a comment here that different `WHERE` clauses are added later in the code, otherwise this query looks a bit ..... big